vochain/scrutinizer/process.go

```diff
@@ -255,15 +255,15 @@ func (s *Scrutinizer) newEmptyProcess(pid []byte) error {
 	// Add the entity to the indexer database
 	eid := p.GetEntityId()
 	entity := &Entity{}
-	// If entity is not registered in db, add to entity count cache
+	// If entity is not registered in db, add to entity count cache and insert to db
 	if s.db.FindOne(entity, badgerhold.Where("ID").Eq(eid)); entity == nil {
```

Comment:
`s.db.FindOne` returns a "badgerhold.ErrNotFound" if the record does not exist, that should be used instead of `engity == nil`. Also if the error is another, return it.

In general no error can be unattended.